docs/security/faq.md

```diff
@@ -0,0 +1,188 @@
+The gist of it is that it's extremely unlikely for an attacker to gain access to the LND database without also gaining access to other sensitive information.
+
+If you don't use unlocker, you risk LND being down for too long and your peer stealing your sats.
+LND being down is much more likely to happen.
+
+The developers of LND themselves agreed recently that allowing empty passwords makes sense.
+Read the issue for in-depth explanation.
+
+## There are so many apps! Don't they increase attack surface?
+
+They do if you install all of them.
+The point of the project is that you only install what you require.
+If you don't trust some app and don't install it, the attack surface of it is zero.
+
+## I ran systemd-analyze security and there's a bunch of services with bad reports!
+
+The short answer is that `systemd-analyze security` is a huge crap.
+This is even admitted in the manual although not this honestly.
+
+The specific objections to it:
+
+* Some security options imply others are not needed.
+  As an example, if the process is not `root` and can not become `root`, it's impossible for it to load kernel modules.
+  Yet, `systemd-analyze security` reports that it is possible and assings it security score.
+  Similar thing applies to other flags.
+* Some security options have zero real effect on security.
+  The worst example is restriction by IP address.
+  IP addresses can be spoofed by trained monkeys, so implementing restrictions based on them is just a waste of time and introduces unneeded complexity.
+* Some options, especially `PrivateNetwork` are assigned high risk but they destroy fundamental features of a service.
+  `bitcoind` without network access is completely useless.
+
+Yes, having some kind of ordering to help decide on priority of security review would be useful, especially regarding the last point.
+Sadly, this tool doesn't even order its output
+and tries to emotionally manipulate people into setting ridiculous options,
+wasting time on non-issues, writing explanations like this one, or worst, distrusting something that's actually pretty secure.
+
+This doesn't intend to imply that all relevenat options are tuned perfectly.
+If you know of a reasonable improvement, please report!
+Using `systemd-analyze security` is just useless because of lot of noise.
+
+## Why is LND seed stored in plaintext?
+
+The same reason why there's unlocker.
+The seed is stored in a protected file in private directory.
+If an attacker can obtain the seed, he can obtain the admin macaroon and and tls key.
+These files can not be encrypted as it'd make LND unusable.
+
+If you're concerned about people with physical access stealing your node and sats, just encrypt the whole OS.
+
+## Are there automated (channel) backups?
+
+Not yet, they are planned and considered high priority.
+
+## How are Bitcoin forks handled?
+
+Please read the documentation in the top-level README.
```

Comment:
link that's consistent with this PR?

Reply:
I'm still undecided if fork policy should go to its own document. Probably yes, but should be promoted.

Reply:
ACK for separate doc